cdc/processor/metrics.go

```diff
@@ -60,6 +60,13 @@ var (
 			Name:      "exit_with_error_count",
 			Help:      "counter for processor exits with error",
 		}, []string{"changefeed", "capture"})
+	processorSchemaStorageGcTsGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "ticdc",
+			Subsystem: "processor",
+			Name:      "gc_ts",
```

Comment:
Could you add a prefix?
```suggestion
			Name:      "schema_storage_gc_ts",
```